server.js

```diff
@@ -58,6 +58,11 @@ app.get("/auspice*", (req, res) => {
   res.redirect('https://nextstrain.github.io/auspice/');
 });
 
+/* handle redirects for inrb-drc (first of the Nextstrain groups) */
+app.get("/inrb-drc*", (req, res) => {
+  res.redirect(`/groups${req.path}`);
```

Comment:
This redirect doesn't preserve query parameters, which are used to define parts of the Auspice visual state. For example, going to `/inrb-drc/ebola/2019-12-09?c=num_date` redirects to `/groups/inrb-drc/ebola/2019-12-09`.

Reply:
So _that's_ the difference between `req.path` and `req.originalUrl`. Got it. Thanks.